src/app/ui/font_entry.cpp

```diff
@@ -430,14 +476,45 @@ void FontEntry::onStyleItemClick(ButtonSet::Item* item)
   switch (m_style.getItemIndex(item)) {
     // Bold button changed
     case 0: {
-      const bool bold = m_style.getItem(0)->isSelected();
-      style = text::FontStyle(
-        bold ? text::FontStyle::Weight::Bold : text::FontStyle::Weight::Normal,
-        style.width(),
-        style.slant());
+      if (m_availableWeights.size() > 1) {
```

Comment:
> What used to take 1 click (setting the font to bold), now takes 2 clicks. 

To get the 1 click back in the case of having just 2 weights available:

```suggestion
      if (m_availableWeights.size() > 2) {
```